index.js

```diff
@@ -2,81 +2,76 @@ module.exports = {
   env: {
     es6: true,
     node: true,
-    mocha: true
+    mocha: true,
   },
   plugins: ['mocha', 'prettier'],
-  extends: ['eslint-config-airbnb', 'prettier'],
+  extends: ['airbnb', 'prettier'],
   rules: {
-    // http://eslint.org/docs/rules/no-use-before-define
-    'no-use-before-define': [
-      2,
-      {
-        functions: false
-      }
-    ],
-
     // http://eslint.org/docs/rules/quotes
     quotes: [
       2,
       'single',
       {
         avoidEscape: true,
-        allowTemplateLiterals: true
-      }
+        allowTemplateLiterals: true,
+      },
     ],
 
     // http://eslint.org/docs/rules/comma-dangle
-    'comma-dangle': [2, 'never'],
-
-    // http://eslint.org/docs/rules/no-shadow
-    'no-shadow': [0],
-
-    // http://eslint.org/docs/rules/no-param-reassign
-    'no-param-reassign': [2, { props: false }],
+    'comma-dangle': [2, 'always-multiline'],
 
     // http://eslint.org/docs/rules/strict
     strict: [0],
 
     'mocha/no-exclusive-tests': 'error',
 
-    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md
-    'react/prefer-stateless-function': [1],
-
-    'react/require-extension': 'off',
-
     'generator-star-spacing': [
       'error',
       {
         before: true,
-        after: false
-      }
+        after: false,
+      },
     ],
 
-    'jsx-a11y/href-no-hash': 'off',
-
+    // http://eslint.org/docs/rules/no-shadow
+    // http://eslint.org/docs/rules/no-param-reassign
+    // http://eslint.org/docs/rules/no-use-before-define
+    'no-use-before-define': [
+      2,
+      {
+        functions: false,
+      },
+    ],
+    'no-param-reassign': [2, { props: false }],
+    'no-unused-expressions': 'off',
+    'no-confusing-arrow': 'off',
```

Comment:
this also a good rule to have

```js
const a = b => c ? 1 : 2;
```

you can't tell whether it's boolean `b => c` or it's a function call `(b) => { return c ? 1 : 2 }`

i think this would be an option

```
"no-confusing-arrow": ["error", {"allowParens": false}]
```